src/components/Description.js

```diff
@@ -29,14 +31,18 @@ function Description() {
                   <path d='M8 17l4 4 4-4m-4-5v9'></path>
                   <path d='M20.88 18.09A5 5 0 0018 9h-1.26A8 8 0 103 16.29'></path>
                 </svg>
-                <h2 className='title-font font-medium text-3xl text-gray-800'>
-                  Decentralized Solution
+                <h2 className=' font-medium text-gray-900'>
+                  Configurable Rules
                 </h2>
-                <p className='leading-relaxed'>Discover</p>
+                <p className='text-gray-700 mt-4 flex-1'>
```

Comment:
I think we need to keep the sizes fixed for the cards. It looks uneven. Thoughts on this?

Reply:
yes, it's uneven and the card last one is left aligned. this was because of lengthy class names. Flex property was not working .will add the icons clean this up.